cmake/environmentchecks.cmake

```diff
@@ -84,25 +84,25 @@ else()
 endif()
 
 if(CLANG_FORMAT_EXE AND GIT_EXE)
-  set(CLANG_FORMAT_TESTING_AVAILABLE TRUE INTERNAL BOOL "Executables required to run the ClangFormat test are available.")
+  set(CLANG_FORMAT_TESTING_AVAILABLE TRUE CACHE INTERNAL "Executables required to run the ClangFormat test are available.")
```

Comment:
just to check, `INTERNAL BOOL` is just wrong, right? It doesn't actually define a boolean type variable? And the implication of `INTERNAL` being a string-valued variable has no affect on the way we're using this? (or am I reading the docs wrong?)

Reply:
Yes `INTERNAL BOOL` is just wrong.  Being string valued has no affect on the way we're using this.  The string still evaluates to true, it's just not a constant true as it would be if it were a boolean.